### PR TITLE
Learning functions

### PR DESCRIPTION
**Pull request by ioandodita069**

### CODE REVIEW

**Comment by ioandodita069** [comment]
Screenshot from 2022-04-27 18-31-06

**Comment by ioandodita069** [comment]
!
Screenshot from 2022-04-27 18-41-05

Screenshot from 2022-04-27 18-40-47

Screenshot from 2022-04-27 18-39-17
ttps://user-images.githubusercontent.com/94746070/165
Screenshot from 2022-04-27 18-40-13
557571-ed17c50d-50e3-4789-bf81-e74676caf397.png)
[Screenshot from 2022-04-27 18-33-51
Screenshot from 2022-04-27 18-39-39

Screenshot from 2022-04-27 18-39-39
ser-images.githubusercontent.com/94746070/165557255-e4aedee6-a11e-447b-b73f-fcf478ee792a.png)

Screenshot from 2022-04-27 18-37-52
![Screenshot from 2022-04-27 18-38-55

Screenshot from 2022-04-27 18-39-17
](https://u
Screenshot from 2022-04-27 18-37-05
ser-images.githubusercontent.com/94746070/165556872-8a8d972e-f929-410c-ac90-188e5d0e96da.png)
](https://us
Screenshot from 2022-04-27 18-37-27

Screenshot from 2022-04-27 18-36-36

Screenshot from 2022-04-27 18-35-04
er-images.githubusercontent.com/94746070/1
Screenshot from 2022-04-27 18-34-17
65556316-858690b5-4456-444e-b39a-59e54ca66e3b.png)
65556294-7594d8ad-907a-43d5-8cfa-84c59b635ee6.png)

File: js/array-sort.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    let len = arr.length;
+    let max = -Infinity;
+    while (len--) {
+        if (arr[len] > max) {
+            max = arr[len];
+        }
+    }
+    return max;
+}
+console.log(highestNumber(points))
```
Comment (pancudaniel7) [inline js/array-sort.js:13]: Good!

**Review by pancudaniel7 — APPROVED** [review]
Looks better!

**Comment by danieljecu** [comment]
the code is looking good

**Pull request merged**